lib.rs

```diff
@@ -63,14 +64,16 @@ fn level_to_severity(level: slog::Level) -> syslog::Severity {
 pub struct Streamer3164 {
     io: Mutex<Box<syslog::Logger>>,
     format: Format3164,
+    level: Level,
 }
 
 impl Streamer3164 {
     /// Create new syslog ``Streamer` using given `format`
-    pub fn new(logger: Box<syslog::Logger>) -> Self {
+    pub fn new(logger: Box<syslog::Logger>, level: Level) -> Self {
```

Comment:
This would a backward-incompat. change. Do we need to bump the version? Alternatively, you could introduce a new method, and leave this one as is.

Reply:
I think it would be better if I were to introduce a new method and leave that one as it is, with a documented sane default log level(maybe one that has something to do with the slog feature for max log level for debug and release builds).

Reply:
Regarding the version bump, I noticed there is a TODO, this issue https://github.com/slog-rs/slog/issues/261 and a missing `Streamer5424`. Does this library accept contributions? I'd like to look into those issues myself, if there is nobody else on the task.

Reply:
I am always happy to accept PRs.